packages/dnb-eufemia/src/components/accordion/Accordion.tsx

```diff
@@ -228,13 +228,17 @@ function Accordion({
     if (context?.expanded_id && context.expanded_id === props.id) {
       setExpanded(true)
     }
-  }, [context.flush_remembered_state, context.expanded_id])
+  }, [
+    context.flush_remembered_state,
+    context.expanded_id,
+    props.expanded,
+    props.id,
```

Comment:
```suggestion
id,
```

Is there a reason we don't use the deconstructed props like this?

Reply:
Nope 😄 We should propably clean that up in a seperate pr, since there seem to be a lot of those occurances